lib/rules/max-len.js

```diff
@@ -157,11 +157,14 @@ module.exports = {
          */
         function isFullLineComment(line, lineNumber, comment) {
             var start = comment.loc.start,
-                end = comment.loc.end;
+                end = comment.loc.end,
+                isFirstTokenOnLine = !line.slice(0, comment.loc.start.column).trim(),
```

Comment:
Reusing this from the no-inline-comments rule https://github.com/eslint/eslint/blob/cd1b0574f1af79d189259f4e1a255230d24b37ee/lib/rules/no-inline-comments.js#L39